colin-api/src/colin_api/models/filing.py

```diff
@@ -1805,11 +1804,6 @@ def _process_ar(cls, cursor, filing: Filing, corp_num: str, ar_date: str, agm_da
                 if agm_year >= last_year:
                     Business.update_corp_state(cursor=cursor, event_id=filing.event_id, corp_num=corp_num)
 
-            # create new ledger text for annual report
-            ledger_year = agm_date if agm_date else f'NO AGM HELD IN {agm_year}'
```

Comment:
This comment is not required. agm_date is required in COOP and we don't ask for CORPS